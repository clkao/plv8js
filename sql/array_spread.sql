do $$
  Object.prototype[Symbol.iterator] = function() {
     return {
       next:() => this
     }
  };
  [...({})];
$$ language plv8;
do $$
  Object.prototype[Symbol.iterator] = function() {
     return {
       next:() => this
     }
  };
  [...({})];
$$ language plv8;
