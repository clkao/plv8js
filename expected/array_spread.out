do $$
  Object.prototype[Symbol.iterator] = function() {
     return {
       next:() => this
     }
  };
  [...({})];
$$ language plv8;
ERROR:  Out of memory error
do $$
  Object.prototype[Symbol.iterator] = function() {
     return {
       next:() => this
     }
  };
  [...({})];
$$ language plv8;
ERROR:  Out of memory error
